elements/nvidia-cuda/post-install.d/95-nvidia-cuda

```diff
@@ -3,8 +3,11 @@
 if [ "$NVIDIA_CUDA_DELETE_REPO" = "y" -o "$NVIDIA_CUDA_DELETE_REPO" = "Y" ]
 then
     case "$DISTRO_NAME" in
-	centos* | rhel* | fedora*)
-	    rm /etc/yum.repos.d/cuda.repo
+	centos* | rhel*)
+	    rpm -e cuda-repo-rhel7-${NVIDIA_CUDA_VERSION}.x86_64
+	    ;;
+	fedora*)
+	    rpm -e cuda-repo-fedora25-${NVIDIA_CUDA_VERSION}.x86_64
```

Comment:
Use $NVIDIA_CUDA_DISTRO?

Reply:
Good spot, that tidies up well.  I'll make that fix and push in.